packages/esm-patient-forms-app/src/config-schema.ts

```diff
@@ -32,31 +32,31 @@ export const configSchema = {
       {
         formUuid: 'd2c7532c-fb01-11e2-8ff2-fd54ab5fdb2a',
         formName: 'Admission (Simple)',
-        formAppUrl: 'simpleAdmission',
+        formAppUrl: 'referenceapplication:htmlforms/simpleAdmission.xml',
```

Comment:
We may want to rename the `formAppUrl` config element since it is not actually a URL. Maybe `formUiResource`. Though I'm not sure it's useful to have all these names prefixed with `form` the way they are.